content/docs/command-reference/commit.md

```diff
@@ -62,26 +61,25 @@ There are many cases where the last step is not desirable (for example rapid
 iterations on an experiment). The `--no-commit` option prevents the last step
 from occurring (on the commands where it's available), saving time and space by
 not storing unwanted <abbr>data artifacts</abbr>. The file hash is still
-computed and added to the DVC-file, but the actual data file is not saved in the
-cache. This is where the `dvc commit` command comes into play. It performs that
-last step (saving the data in cache).
+computed and added to `dvc.lock` or `.dvc` file, but the actual data file is not
+saved in the cache. This is where the `dvc commit` command comes into play. It
+performs that last step (saving the data in cache).
 
 Note that it's best to avoid the last two scenarios. They essentially
-force-update the [DVC-files](/doc/user-guide/dvc-files-and-directories) and save
-data to cache. They are still useful, but keep in mind that DVC can't guarantee
-reproducibility in those cases.
+force-update the DVC-tracked files and save data to cache. They are still useful
+, but keep in mind that DVC can't guarantee reproducibility in those cases.
```

Comment:
"force-update the DVC-tracked files" is incorrect 🙁 

And no spaces before commas `,` please.

Reply:
The `,` was in new line because of 80 character limit probably.

Reply:
Negative. The formatter doesn't do that, the whole last word including comma would  be put underneath. This was a manual typo.

Reply:
Yeah I think error was from my side. I didn't realize adding `,` to next line will also add a space before it. Word plus `,` both will come in next line